receiver/windowseventlogreceiver/README.md

```diff
@@ -88,3 +89,20 @@ Output entry sample:
 }
 ```
 
+#### Remote Configuration
+
+If collection of the local event log is desired, a separate receiver needs to be created.
+
```

Comment:
We need also to include a mention of the requirements on the remote server:

> the remote computer must enable the "Remote Event Log Management" Windows Firewall exception; otherwise, when you try to use the session handle, the call will error with RPC_S_SERVER_UNAVAILABLE. The computer to which you are connecting must be running Windows Vista or later.

Source: https://learn.microsoft.com/en-us/windows/win32/wes/accessing-remote-computers

Reply:
Updated in readme @pjanotti